NuGet.config

```diff
@@ -2,9 +2,8 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="arcade" value="https://dotnetfeed.blob.core.windows.net/dotnet-tools-internal/index.json" />
-    <add key="roslyn-tools" value="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
```

Comment:
we need to keep `roslyn-tools` for this repo

Reply:
Is it taking regular updates from that repo? If it's one-off stuff I can probably push those packages. We may lose myget sometime in the future.

Reply:
this is the package: https://dotnet.myget.org/feed/roslyn-tools/package/nuget/UpdateTemplateVersion

I do not update it regularly. We can move it whereever

Reply:
You're uploading manually? If so, then I think it's pretty trivial to just upload to dotnet-eng instead (you should have permissions) and thus we can deprecate this feed.

Reply:
never done an upload to dnceng before. myget was two clicks all told. Where are the docs for dnceng packages?

Reply:
https://dev.azure.com/dnceng/public/_packaging?_a=connect&feed=dotnet-eng

Once you install the credential provider for AzDO, it's even simpler, and it auths for you.

```
nuget push --source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json <package>
```

Reply:
Sorry:

```
dotnet nuget push --source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json --api-key az <package-path>
```

Reply:
```cmd
λ dotnet nuget push --source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json --api-key az UpdateTemplateVersion.1.0.4-beta1.nupkg
Pushing UpdateTemplateVersion.1.0.4-beta1.nupkg to 'https://pkgs.dev.azure.com/dnceng/9ee6d478-d288-47f7-aacc-f6e6d082ae6d/_packaging/1a5f89f6-d8da-4080-b15f-242650c914a8/nuget/v2/'...
  PUT https://pkgs.dev.azure.com/dnceng/9ee6d478-d288-47f7-aacc-f6e6d082ae6d/_packaging/1a5f89f6-d8da-4080-b15f-242650c914a8/nuget/v2/
  Unauthorized https://pkgs.dev.azure.com/dnceng/9ee6d478-d288-47f7-aacc-f6e6d082ae6d/_packaging/1a5f89f6-d8da-4080-b15f-242650c914a8/nuget/v2/ 456ms
error: Response status code does not indicate success: 401 (Unauthorized).
```

Says I am not authorized, I followed the directions to install the credential provider but it does not change the output.